components/script/dom/shadowroot.rs

```diff
@@ -642,6 +643,11 @@ impl<'dom> LayoutShadowRootHelpers<'dom> for LayoutDom<'dom, ShadowRoot> {
         unsafe { &self.unsafe_get().author_styles.borrow_for_layout().data }
     }
 
+    #[inline]
+    fn is_ua_widget(&self) -> bool {
+        self.unsafe_get().is_user_agent_widget()
```

Comment:
Is it Ok to use insafe without #[expect(unsafe_code)] + unsafe { ..} ?

Reply:
https://github.com/servo/servo/blob/660238a94c417d709cc99ec3d6cc6d0bfe4fc272/components/shared/layout/wrapper_traits.rs#L196-L198